### PR TITLE
Removed obsolete lines, added all the missing ones, fixed some old ones.

### PR DESCRIPTION
**Pull request by Viandante**

### CODE REVIEW

File: CorsixTH/Lua/languages/italian.lua [hunk 1]
```diff
@@ -119,14 +142,25 @@ menu_debug_overlay = {
...
+}
 adviser = {
   room_forbidden_non_reachable_parts = "Mettere la stanza in questa posizione risulterebbe nel blocco dell'accesso ad alcune parti dell'ospedale.",
   warnings = {
     no_desk = "Prima o poi dovrai costruire un banco di accettazione ed assumere una receptionist!",
     no_desk_1 = "Se vuoi che i pazienti vengano nel tuo ospedale dovrai assumere una receptionist e costruirle una scrivania dove farla lavorare!",
     no_desk_2 = "Ben fatto, deve essere un record mondiale: quasi un anno e nessun paziente! Se vuoi continuare come Manager di questo ospedale dovrai assumere una receptionist e costruire un banco di accettazione dove farla lavorare!",
     no_desk_3 = "Grandioso, quasi un anno e ancora non hai personale all'accettazione! Come ti aspetti di trovare pazienti, ora vedi di rimediare e di non fare più casini!",
-    cannot_afford = "Non hai abbastanza soldi in banca per assumere questa persona!", -- I can't see anything like this in the original strings
+	no_desk_4 = "Una receptionist ha bisogno della sua stazione di lavoro per accogliere i clienti quando arrivano",
```
Comment (TheCycoONE) [inline CorsixTH/Lua/languages/italian.lua]: should be spaces not tabs

File: CorsixTH/Lua/languages/italian.lua [hunk 2]
```diff
@@ -119,14 +142,25 @@ menu_debug_overlay = {
...
+    no_desk_5 = "Beh, era ora, presto dovresti cominciare a vedere arrivare i pazienti!",
+    no_desk_6 = "Hai una receptionist, che ne dici quindi di costruirle un banco di accettazione dove poter lavorare?",
+    no_desk_7 = "Hai costruito il banco di accettazione, quindi che ne dici di assumere una receptionist? Non vedrai pazienti finché non risolvi questa cosa!",
+    cannot_afford = "Non hai abbastanza soldi in banca per assumere quella persona!", -- I can't see anything like this in the original strings
+	cannot_afford_2 = "Non hai abbastanza soldi in banca per fare quell'acquisto!",
```
Comment (TheCycoONE) [inline CorsixTH/Lua/languages/italian.lua]: spaces

File: CorsixTH/Lua/languages/italian.lua [hunk 3]
```diff
@@ -135,23 +169,27 @@ adviser = {
     falling_6 = "Non è una sala da bowling, i malati non dovrebbero essere trattati così!",
     research_screen_open_1 = "Devi costruire un Reparto Ricerca prima di poter accedere alla schermata Ricerca.",
     research_screen_open_2 = "La Ricerca è disabilitata in questo livello.",
+	researcher_needs_desk_1 = "Un ricercatore ha bisogno di una scrivania su cui lavorare.",
```
Comment (TheCycoONE) [inline CorsixTH/Lua/languages/italian.lua]: spaces

File: CorsixTH/Lua/languages/italian.lua [hunk 4]
```diff
@@ -417,19 +557,23 @@ cheats_window = {
...
+    yes = "Cheat usati: Sì",
   },
   cheats = {
     money = "Cheat soldi",
-    all_research = "Cheat Completa Ricerche",
+    all_research = "Completa Ricerche",
     emergency = "Crea Emergenza",
     vip = "Crea VIP",
     earthquake = "Crea Terremoto",
+	epidemic = "Genera paziente contagioso",
```
Comment (TheCycoONE) [inline CorsixTH/Lua/languages/italian.lua]: spaces

File: CorsixTH/Lua/languages/italian.lua [hunk 5]
```diff
@@ -417,19 +557,23 @@ cheats_window = {
...
+	toggle_infected = "Attiva o disattiva icone infetti",
     create_patient = "Crea Paziente",
     end_month = "Fine Mese",
     end_year = "Fine Anno",
     lose_level = "Perdi Livello",
     win_level = "Vinci Livello",
+	increase_prices = "Aumenta prezzi",
```
Comment (TheCycoONE) [inline CorsixTH/Lua/languages/italian.lua]: spaces

File: CorsixTH/Lua/languages/italian.lua [hunk 6]
```diff
@@ -442,11 +586,15 @@ tooltip.cheats_window = {
     emergency = "Crea un'emergenza.",
     vip = "Crea un VIP.",
     earthquake = "Crea un terremoto.",
+	epidemic = "Crea un paziente contagioso che potrebbe causare un'epidemia",
```
Comment (TheCycoONE) [inline CorsixTH/Lua/languages/italian.lua]: spaces

File: CorsixTH/Lua/languages/italian.lua [hunk 7]
```diff
@@ -442,11 +586,15 @@ tooltip.cheats_window = {
...
+	toggle_infected = "Attiva o disattiva le icone infetti per le epidemie attive scoperte",
     create_patient = "Crea un Paziente sul bordo della mappa.",
     end_month = "Salta alla fine del mese.",
     end_year = "Salta alla fine dell'anno.",
     lose_level = "Perdi il livello corrente.",
     win_level = "Vinci il livello corrente.",
+	increase_prices = "Aumenta tutti i prezzi del 50% (max 200%)",
```
Comment (TheCycoONE) [inline CorsixTH/Lua/languages/italian.lua]: spaces

File: CorsixTH/Lua/languages/italian.lua [hunk 8]
```diff
@@ -476,6 +624,52 @@ tooltip.calls_dispatcher = {
...
+}
+
+tooltip.update_window = {
+  download = "Vai alla pagina del download per trovare l'ultima versione di CorsixTH",
+  ignore = "Ignora l'aggiornamento per ora. Questa notifica apparirà di nuovo al prossimo avvio di CorsixTH",
+}
+
+map_editor_window = {
+  pages = {
+    inside = "Interno",
```
Comment (TheCycoONE) [inline CorsixTH/Lua/languages/italian.lua]: spaces

**Comment by TheCycoONE** [comment]
I don't speak Italian, but it looks good other than a handful of tab vs space issues.

**Comment by TheCycoONE** [comment]
Are you ok with amending the commit, or would you like help with this?

**Comment by Viandante** [comment]
Damn, on git shell I couldn't find a way to make "git commit --amend" work, so I tried editing it here but it created another commit.
Should I just close the request and make a new one?

**Comment by TheCycoONE** [comment]
mm... if you can use the command line (git bash) you can still fix it:
```
git rebase -i HEAD~2
in the text file that opens change the last line where it says "pick" to "f"
save/close the text editor
push -f when done
```
If you'd rather not, I can submit on your behalf.

**Comment by Viandante** [comment]
When I tried to do that it picked also the changes made by Grimdoc.
Now the terminal reads "C:\Users\Viandante\Documents\GitHub\CorsixTH [detached HEAD|REBASE-i +1 ~0 -0 !]>", what do I do?

I'd really like to commit it myself to learn a bit about this system. I thought a force push would be a sin, or at least that's what people say on stackOverflow and the likes! :)

**Comment by TheCycoONE** [comment]
Ah... because you did it online the changes weren't in your local repository.

git is a powerful but tricky tool to use, so certainly don't feel you need to master it unless you want to become a coder. We are ok with translations being pasted in a github issue or even submitted to the google group instead.

In the future when using git, do not do any work in the master branch - make a new branch for each pull request. It will save problems down the road.

You were right to be weary of a force push when things didn't look right. Ultimately we will have to do one but we need everything to look good first.

```
git reset --hard master
git pull
git rebase -i HEAD~2
there should be both of your commits listed in the text file here, change the second one's "pick" to "f", save and close
follow rebase instructions
Assuming everything looks good: git push -f
```

**Comment by Viandante** [comment]
Thank you, it seems sorted out now.
I actually tried to create a branch called "ITtranslation" to work on but something went wrong along the way and changes went to master instead.

I guess I'll have to study the system a bit more, I'm very interested in learning.
In the meantime, thanks for your help!

**Comment by Alberth289346** [comment]
# Best is if you are on the master branch, and it is up-to-date with corsix master
git branch ITranslation    # make the branch
git checkout ITranslation  # switch to the branch

It is possible to combine both commands into one with a flag of the "checkout" command.

**Pull request merged**